modules/db_option_group/outputs.tf

```diff
@@ -0,0 +1,10 @@
+# DB option group
+output "this_db_option_group_id" {
+  description = "The db option group id"
+  value       = "${aws_db_option_group.this.id}"
+}
+
+output "this_db_option_group_arn" {
+  description = "The ARN of the db option group"
+  value       = "${aws_db_option_group.this.arn}"
```

Comment:
And this - `value = "${element(split(",", join(",", aws_db_option_group.this.*.arn)), 0)}"`